rholang/examples/vault_demo/0.transfer_from_genesis.rho

```diff
@@ -0,0 +1,60 @@
+// A set of testnet admin-controlled 'genesis vault'-s will be our approach for providing
+// a testnet-token supply for the Testnet 1.
+// Everything regarding the genesis vaults is subject to change.
+
+new
+  rl(`rho:registry:lookup`), revAddress(`rho:rev:address`), RevVaultCh, getDeployData(`rho:deploy:params`),
+  deployerPubKeyCh, vaultCh, getDeployerPubKey,
+  stdout(`rho:io:stdout`), log
+in {
+
+  rl!(`rho:id:1o93uitkrjfubh43jt19owanuezhntag5wh74c6ur5feuotpi73q8z`, *RevVaultCh) |
+  getDeployerPubKey!(*deployerPubKeyCh) |
+  for (@(_, RevVault) <- RevVaultCh; @pubKey <- deployerPubKeyCh) {
+
+    log!(("0.create_genesis_vault.rho")) |
+    log!(("Using public key:", pubKey, "to obtain a `deployer's-pubkey-bound` vault")) |
+
+    @RevVault!("findOrCreateGenesisVault", pubKey, 9000, *vaultCh ) |
+
+    // most Vault methods return an `Either[String, A] = ("Left", String) \/ ("Right", A)`
+    for (@("Right", vault) <- vaultCh) {
+
+      log!(("Genesis vault obtained")) |
+
+      match ("%REV_ADDR", 100) {
```

Comment:
It's already pretty obvious but it might add some extra clarity if you use `%RECIPIENT_REV_ADDR` or something like that.

Reply:
I'd rather not, it's easier to type during the demo that way :)